migrations/subfolderrights.js

```diff
@@ -0,0 +1,56 @@
+/* eslint no-console: 0 */
+/* eslint no-confusing-arrow: 0 */
+const ran = false; // set to true to exclude migration
+const name = 'Fix access rights for teachers on homework submitted files.';
+
+const database = require('../src/utils/database');
+
+const RoleModel = require('../src/services/role/model');
+const { FileModel } = require('../src/services/fileStorage/model.js');
+
+
+const run = async (dry) => {
+	database.connect();
+
+	const errorHandler = (e) => {
+		console.log('Error', e);
+		return undefined;
+	};
+
+	const { _id: studentRoleId } = await RoleModel.findOne({ name: 'student' }).lean().exec();
+
+	const courseFolders = await FileModel
+		.find({
+			isDirectory: true,
+			refOwnerModel: 'course',
+		})
+		.exec()
```

Comment:
`.lean().exec()` should give us more performance here, since we do not need the mongoose document (only the data)